cves/kernel/CVE-2017-7487.yml

```diff
@@ -129,16 +128,16 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: No unit testing was used to find this vulnerability
+  fix: false
+  fix_answer: The fix had no unit testing involved
```

Comment:
How did you come to the conclusion that there wasn't any unit testing? What sources did you check?

Reply:
Added conclusion source